app/reactiveLayer/behaviorManager/conf/default.ini

```diff
@@ -2,7 +2,7 @@ use_ears        false
 
 [BEHAVIORS]
 
-behaviors	(tagging, pointing, moveObject, ask, followingOrder, recognitionOrder)
+behaviors	(tagging, pointing, moveObject)
 ttsOptions      "iCub"
 
 [followingOrder]
```

Comment:
Hey @jypuigbo,
One last question before merging. Should `followingOrder` really be removed? Will we still be able to say "Point to the octopus", where octopus is an unknown object? Is this handled by `tagging`? If not, we should keep `followingOrder`
If `followingOrder` is to be removed, we should remove all references to it, i.e. the `[followingOrder]` group in `app/reactiveLayer/behaviorManager/conf/default.ini` and the reference to "followingOrder" in `ears.cpp` Line 221/226.

Best, Tobi